image/tools/lib/component/resources.sh

```diff
@@ -13,7 +13,7 @@ function check_resource {
     # was returned but no actual results. That would be at
     # least two lines: one for the header and one for each
     # resource found
-    if [ "$result"  -eq "1" ]; then
+    if [[ "$result"  -eq "1" ]]; then
```

Comment:
any reason you had to change it?

Reply:
My IDE suggests this, no immediate reason though

Reply:
The previous way (one []) is posix standard which enables people with non linux machines to run those scripts locally (such as macs) - could you please revert it to the old way?